src/Routing.js

```diff
@@ -39,55 +39,64 @@ export default function Routing({ appProps }) {
       Component: Overview,
       path: '/dashboard',
       allowedIf: userIsOfficerOrAdmin,
-      redirect: '/'
+      redirect: '/',
+      InAdminNavbar: true
     },
     {
       Component: EventManager,
       path: '/event-manager',
       allowedIf: userIsOfficerOrAdmin,
-      redirect: '/'
+      redirect: '/',
+      InAdminNavbar: true
     },
     {
       Component: SolidsConsole,
       path: '/3DConsole',
       allowedIf: userIsOfficerOrAdmin,
-      redirect: '/'
+      redirect: '/',
+      InAdminNavbar: true
     },
     {
       Component: LedSign,
       path: '/led-sign',
       allowedIf: userIsOfficerOrAdmin,
-      redirect: '/'
+      redirect: '/',
+      InAdminNavbar: true
     },
     {
       Component: PrintingSolids,
       path: '/3DPrintingForm',
       allowedIf: userIsMember || userIsOfficerOrAdmin,
-      redirect: '/login'
+      redirect: '/login',
+      InAdminNavbar: false
```

Comment:
Instead of explicitly defining them as false, you can just omit the field when you dont need it.

[on this line](https://github.com/SCE-Development/Core-v4/blob/fe1b1d56d8bffd86e5a41425bdb11fa59bf0bded/src/Components/Navbar/NavBarWrapper.js#L19) we check if we should enable the navbar or not, and it will work the same if we do
```suggestion
```